lib/Dancer2/Manual.pod

```diff
@@ -73,12 +80,52 @@ Create a web application using the dancer script:
     + MyApp/views/layouts
     + MyApp/views/layouts/main.tt
 
-It creates a directory named after the name of the app, along with a
-configuration file, a views directory (where your templates and layouts
-will live), an environments directory (where environment-specific
-settings live), a module containing the actual guts of your application, and
-a script to start it. A default skeleton is used to bootstrap the new
-application, but you can use the C<-s> option to provide another skeleton.
+The L<dancer2> script creates the following:
+
+=over
+
+=item *
+a directory named after the name of the app, L<MyApp> in this case
+
+=item *
+
+a configuration file, l<MyApp/config.yml>.
```

Comment:
I think you meant `L<>`, not `l<>`. In all actuality, I think you need `F<>` since you are indicating a file name. There are a few more instances of this below.